projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -133,10 +135,11 @@ import {
     </div>
   `
 })
-export class ExplorerComponent {
+export class ExplorerComponent implements OnDestroy {
   private static readonly VISUALIZATION_EXPANDED_PREFERENCE: string = 'explorer.visualizationExpanded';
   private static readonly RESULTS_EXPANDED_PREFERENCE: string = 'explorer.resultsExpanded';
   private readonly explorerDashboardBuilder: ExplorerDashboardBuilder;
+  private readonly subscriptions: Subscription = new Subscription();
```

Comment:
@cseas Instead of creating a new subscription you can use the `SubscriptionLifecycle` service. Refer - https://github.com/razorpay/hypertrace-ui/blob/rzp_main/projects/observability/src/pages/apis/service-detail/apis/service-apis-list.component.ts

Reply:
Not sure what's happening in the file linked. What we're passing to the subscriptionLifecycle service isn't actually a `subscribe` block, so what's going on?

https://github.com/razorpay/hypertrace-ui/blob/rzp_main/projects/observability/src/pages/apis/service-detail/apis/service-apis-list.component.ts#L25

Reply:
Good fine. Done ✅